Validate ftp mirror repo in remote_ssh_target_ftp scenario

b10n1k opened the pull request:
Schedule validate_mirror_repos in remote_ssh_target_ftp

- Related ticket: https://progress.opensuse.org/issues/69220
- Verification run: http://aquarius.suse.cz/tests/2959#step/validate_mirror_repos/6

b10n1k commented:
> Changes look fine, but your VR has failed with an error. Could you please take a look? Maybe module will require more changes.

If you take a look on the progress ticket i have reported a bug for it. in short the URI differs

rwx788 commented:
> > Changes look fine, but your VR has failed with an error. Could you please take a look? Maybe module will require more changes.
> 
> If you take a look on the progress ticket i have reported a bug for it. in short the URI differs

Progress ticket url is wrong one (https://progress.opensuse.org/issues/59220) please, update the description with correct one.

rwx788 commented:
> > Changes look fine, but your VR has failed with an error. Could you please take a look? Maybe module will require more changes.
> 
> If you take a look on the progress ticket i have reported a bug for it. in short the URI differs

I guess problem is that we type directory with leading '/', so I don't think it's a product bug. I believe we should fix the test not to put leading slash before the directory or at least adjust the expectations.

b10n1k commented:
I added another commit to fix this. I have updated the VR at the top and additional VR for last commit  http://aquarius.suse.cz/tests/2960 against remote_vnc
